Benkimeric/MnE-Frontend
# Feat: Implement logout and dynamic sidebar

#### What does this PR do?

#### Description of Task to be completed?
- [x] implement logout
- [x] make sidebar dynamic
- [x] add authorization to the sidebar and pages

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ft-dynamic-sidebar`
- Install all dependencies `yarn install`
- Run  `yarn start`
- Click on the sidebar items to change the pages
- To test authentication, set yourself as only `Administrator`. you won't be able to see the roles section

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Questions:
